src/mixins/nft.js

```diff
@@ -252,14 +252,12 @@ export default {
       return wallet => (wallet === this.iscnOwner ? 'creator' : 'collector');
     },
     nftDisplayState() {
-      if (
-        this.getNFTClassFeaturedSetByAddress(this.getAddress)?.has(this.classId)
-      ) {
+      const address =
+        this.$route.name === 'id' ? this.$route.params.id : this.getAddress;
+      if (this.getNFTClassFeaturedSetByAddress(address)?.has(this.classId)) {
```

Comment:
`this.wallet` is the address
```suggestion
      if (this.getNFTClassFeaturedSetByAddress(this.wallet)?.has(this.classId)) {
```

Reply:
Which mixin should we put a factory function in to let two pages import? Portfolio mixin?